repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -410,17 +411,28 @@ const getMultiServiceList = (
                                             ...getProfileRef(userPeriodTicket),
                                         },
                                         prices: {
-                                            TimeIntervalPrice: {
-                                                version: '1.0',
-                                                id: `op:${product.productName}@${salesOfferPackage.name}@service`,
-                                                Amount: { $t: `${salesOfferPackage.price || product.productPrice}` },
-                                                TimeIntervalRef: {
-                                                    version: '1.0',
-                                                    ref: `op:Tariff@${
-                                                        product.productName
-                                                    }@${product.productDuration.replace(' ', '-')}`,
-                                                },
-                                            },
+                                            TimeIntervalPrice:
+                                                'productDuration' in product
+                                                    ? {
+                                                          version: '1.0',
+                                                          id: `op:${product.productName}@${salesOfferPackage.name}@service`,
+                                                          Amount: {
```

Comment:
Why are we only adding a TimeIntervalPrice when there is a productDuration now?

Reply:
as the timeinterval is not applicable to flatFare multiOp ticket and not present also in the flatFareGeoZone.xml